chapters/synchronization.txt

```diff
@@ -3999,6 +3985,36 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * The pname:srcAccessMask member of each element of
+    pname:pMemoryBarriers must: only include access flags that are
+    supported by any of the pipeline stages in pname:srcStageMask, as
```

Comment:
I'd prefer "one or more of" vs. "any of" here and below. The latter just reads weirdly to my brain.

Reply:
Above I was suggesting "at least one pipeline stage in"

Reply:
I went with "one or more" because it was shorter. Fixed now.

Reply:
??
"one or more of the pipeline stages in"
"at least one pipeline stage in" <==>